### PR TITLE
Add consent

### LINKED ISSUE #767: [Feature Request] - SMS Integration to gather consent from volunteers
### Subject of the issue
SMS to be sent to all volunteers. SMS contents
1. Acknowledge the help with volunteer name.
2. Unique Shortened URL to seek an opinion from volunteer to find his current status.

URL should be unique and should direct to a poll page to indicate the status of the volunteer. The status may be tagged as Active /Inactive. The poll may work in such a way that once a volunteer clicks the link, he is registered as an active volunteer in the system.

### Your environment
REST API offered by KSITM for SMS Gateway.

### PR DESCRIPTION
**Pull request by bjacobt**
### Issue Reference
This PR addresses the Issue : Fixes #767 

### Summarize
Added
1. Page to confirm/cancel volunteer opt-in
2. has_consented flag in db

Please add malayalam test in ./templates/mainapp/volunteer_consent_success.html

Screenshots
**Register as Volunteer with disclaimer**
register_as_volunteer

**Consent page**
consent_page

**Consent success
consent_success
**

### CODE REVIEW

**Comment by shajith** [comment]
@bjacobt could you leave some instructions to test this locally, in the absence of SMS?

**Comment by shajith** [comment]
@bjacobt need to check the incoming ID/ts to confirm which volunteer to apply this to

File: mainapp/views.py [hunk 1]
```diff
@@ -687,10 +686,18 @@ def __init__(self, *args, **kwargs):
         if self.data == {}:
             self.queryset = self.queryset.none()
 
+class VolunteerConsent(UpdateView):
+    model = Volunteer
+    fields = ['has_consented']
+    success_url = '/consent_success/'
+
```
Comment (cibingeorge) [inline mainapp/views.py]: Did we miss the actual code? or theres some django magic going on here

**Comment by cibingeorge** [comment]
Think we need to validate the last 4 digits of timestamp

File: mainapp/views.py (hunk 1 above)
Comment (biswaz) [reply mainapp/views.py]: django magic! it's UpdateView

**Comment by shajith** [comment]
lgtm

**Comment by biswaz** [comment]
lgtm, no breaking stuff here

**Pull request merged**